modules/@shopify/checkout-sheet-kit/android/src/main/java/com/shopify/reactnative/checkoutsheetkit/CustomCheckoutEventProcessor.java

```diff
@@ -142,10 +142,18 @@ public void onStart(@NonNull CheckoutStartEvent event) {
     }
   }
 
+  @Override
+  public void onAddressChangeRequested(CheckoutAddressChangeRequestedEvent event) {
```

Comment:
should be `onAddressChangeStart` for consistency, but let me know if you want to do renames in a followup just calling out.